specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/GetResourcePricingByNameVirtualMachines_example.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "api-version": "2023-08-01-preview",
+    "scopeId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/DEMO/providers/Microsoft.Compute/virtualMachines/VM-1",
+    "pricingName": "VirtualMachines"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/DEMO/providers/Microsoft.Compute/virtualMachines/VM-1/providers/Microsoft.Security/pricings/VirtualMachines",
+        "name": "VirtualMachines",
+        "type": "Microsoft.Security/pricings",
+        "properties": {
+          "pricingTier": "Standard",
+          "isEnabled": true,
+          "freeTrialRemainingTime": "PT0S",
+          "enablementTime": "2023-03-01T12:42:42.1921106Z",
+          "subPlan": "P2",
+          "inherited": true,
+          "source": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+          "extensions": [
+            {
```

Comment:
Please add the following extension too:
            {
                "name": "MdeDesignatedSubscription",
                "isEnabled": "False"
            },

Reply:
Update also in specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/ListPricingsWithPlanFilter_example.json file